src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -20668,6 +20668,12 @@ public static void Main()
                 // (6,12): error CS0270: Array size cannot be specified in a variable declaration (try initializing with a 'new' expression)
                 //         int[out var x1] a = null; // fatal syntax error - 'out' is skipped
                 Diagnostic(ErrorCode.ERR_ArraySizeInDeclaration, "[out var x1]").WithLocation(6, 12),
+                // (6,17): error CS0103: The name 'var' does not exist in the current context
```

Comment:
I have to investigate why this has changed. At first I thought it was because I changed binding of ArrayRankSpecifier in Binder_Expression, but I reverted that and it still occurred.

Reply:
Fixed